e2e/test/scenarios/metabot/metabot.cy.spec.ts

```diff
@@ -10,13 +12,15 @@ describe("Metabot UI", () => {
     cy.intercept("POST", "/api/ee/metabot-v3/v2/agent-streaming").as(
       "agentReq",
     );
-    cy.intercept("GET", "/api/session/properties").as("sessionProperties");
+    cy.intercept("GET", "/api/automagic-dashboards/database/*/candidates").as(
+      "xrayCandidates",
+    );
```

Comment:
By the time x-ray candidates appear on the home page, mostly everything is already rendered. Whereas waiting for session properties leaves most of the page blank and can lead to flakes.